samples/csharp/MicrosoftGraphSDK/Assignment.cs

```diff
@@ -133,5 +133,64 @@ public class Assignment
                 throw new GraphException($"PublishAsync call: {ex.Message}", ex, classId, assignmentId);
             }
         }
+
+        /// <summary>
+        /// Deactivate an assignment, changes the state of an educationAssignment from its original draft status to the Inactive status
+        /// </summary>
+        /// <param name="client">Microsoft Graph service client</param>
+        /// <param name="classId">User class id</param>
+        /// <param name="assignmentId">Assignment id in the class</param>
+        /// <returns>EducationAssignment</returns>
+        public static async Task<EducationAssignment> DeactivateAsync(
+            GraphServiceClient client,
+            string classId,
+            string assignmentId)
+        {
+            try
+            {
+                return await client.Education
+                    .Classes[classId]
+                    .Assignments[assignmentId]
+                    .Deactivate.PostAsync(requestConfig =>
+                    {
+                        requestConfig.Headers.Add(
+                            "Prefer", "include-unknown-enum-members");
+                    });
+            }
+            catch (Exception ex)
+            {
+                throw new GraphException($"DeactivateAsync call: {ex.Message}", ex, classId, assignmentId);
+            }
+        }
+
+        /// <summary>
+        /// Get all the assignments from the class
+        /// </summary>
+        /// <param name="client">Microsoft Graph service client</param>
+        /// <param name="classId">User class id</param>
+        /// <param name="headerName">Header Name</param>
+        /// <param name="headerValue">Header Value</param>
+        /// <returns>EducationAssignmentCollectionResponse</returns>
+        public static async Task<EducationAssignmentCollectionResponse> GetAssignmentsAsync(
+            GraphServiceClient client,
+            string classId,
+            string headerName,
+            string headerValue)
+        {
+            try
+            {
+                return await client.Education
+                    .Classes[classId]
+                    .Assignments
+                    .GetAsync(requestConfig => {
+                        requestConfig.Headers.Add(
+                            headerName, headerValue);
+                    });
+            }
+            catch (Exception ex)
+            {
+                throw new GraphException($"GetAssignmentsAsync call: {ex.Message}", ex, classId);
+            }
+        }
```

Comment:
this method is not used in the sample, remove it please

Reply:
removed the unused method